.github/workflows/build-docs-verifier.yml

```diff
@@ -0,0 +1,40 @@
+name: Build docs-verifier
+on:
+  [push, pull_request]
+
+jobs:
+  build:
+    name: Build
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-latest]
+        configuration: [debug, release]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v2.3.4
+        with:
+          fetch-depth: 0
+      - name: Setup .NET
+        uses: actions/setup-dotnet@v1.7.2
+      - name: Try get cached dependencies
+        uses: actions/cache@v2.1.4
+        with:
+          path: ${{ github.workspace }}/.nuget/packages
+          key: ${{ runner.os }}-nuget-${{ hashFiles('**/packages.lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-nuget-
+      - name: Validate dependencies
+        run: dotnet restore actions/docs-verifier --locked-mode --packages ${{ github.workspace }}/.nuget/packages
```

Comment:
Is this line correct? Or should it be 

```
dotnet restore ../../actions/docs-verifier --locked-mode --packages ${{ github.workspace }}/.nuget/packages
```

Reply:
These commands are all run from repo root (i.e, github.workspace). The workflow runs correctly on my fork.